requirements.txt

```diff
@@ -0,0 +1,10 @@
+genesis==0.0.0
```

Comment:
I would remove this, the genesis client is imported via `import genesis` but the PyPI package is named `adc`. That's probably why both of these are in here, but the `genesis` package refers to an unrelated package we don't want.